LiteFarmOrg/LiteFarm
# Lf 2500/reading tab impl

To Test: 

1. update the sensor reading types.
2. check if the reading types are to be displayed on the "Readings" Tab page.

To Test:
1. Click on the sensor in the farm maps. check if it redirects to the "Readings" Tab page.

For more details:
https://lite-farm.atlassian.net/browse/LF-2500
